test/node/bson_test.js

```diff
@@ -2323,4 +2323,37 @@ describe('BSON', function() {
       expect(() => bson.serialize(doc, { checkKeys: true })).to.not.throw()
     });
   });
-});
+
+  describe('UUID serialize', function() {
+    const BSON = Object.assign({}, createBSON());
+    BSON.UUID = require('bson').UUID;
```

Comment:
If we do this instead:

```suggestion
    const BSON = createBSON();
    const UUID = require('bson').UUID;
```

Then we don't need the new object and can in all cases below just do `new UUID()` instead of `new BSON.UUID()` . This would make it clearer that `UUID` is coming from js-bson and isn't also exported from bson-ext.

Reply:
I've updated.